<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WSRV01" name="Email" tests="3" failures="1" timestamp="19 Sep 2016 11:13:23 GMT" time="27.483" errors="0">
  <testcase name="signIn" time="2.83" classname="com.randtronics.dpm.filemanager.executionEngine.Email_Test"/>
  <testcase name="Email_Valid" time="19.717" classname="com.randtronics.dpm.filemanager.executionEngine.Email_Test">
    <failure type="java.lang.AssertionError" message="Test2 failed with parameters: smtpout.asia.secureserver.net,465.0,test@randtronics.com,No,test@randtronics.com,qa@Angler123,null,Default (For Port 25)">
      <![CDATA[java.lang.AssertionError: Test2 failed with parameters: smtpout.asia.secureserver.net,465.0,test@randtronics.com,No,test@randtronics.com,qa@Angler123,null,Default (For Port 25)
	at org.junit.Assert.fail(Assert.java:88)
	at com.randtronics.dpm.filemanager.executionEngine.Email_Test.Email_Valid(Email_Test.java:98)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Email_Valid -->
  <testcase name="signOut" time="0.704" classname="com.randtronics.dpm.filemanager.executionEngine.Email_Test"/>
</testsuite> <!-- Email -->
